### PR TITLE
luminous: ceph-disk create deprecation warnings

### PR DESCRIPTION
**Pull request by alfredodeza**
Fixes http://tracker.ceph.com/issues/22154

### CODE REVIEW

**Comment by alfredodeza** [comment]
jenkins test docs

File: doc/ceph-volume/index.rst [hunk 1]
```diff
@@ -17,6 +17,33 @@ that may have been deployed with ``ceph-disk``.
...
 * :ref:`ceph-volume-simple`
 
+
+Migrating
+---------
+Starting on Ceph version 12.2.2, ``ceph-disk`` is deprecated. Deprecation
+warnings will show up that will link to this page. It is strongly suggested
+that users start consuming ``ceph-volume``.
+
+New deployments
```
Comment (amitkumar50) [inline doc/ceph-volume/index.rst]: NIT: New Deployments

**Comment by wjwithagen** [comment]
@alfredodeza 
Can you make it so that it is possible to suppress this humongous piece of text?
During debugging of all kinds of script I now get a free but useless warning about depreciation.
And unless somebody also writes `ceph-volume zfs`  I would not like this tool to be deprecated.

**Comment by alfredodeza** [comment]
@wjwithagen these warnings are using Python's `warning` module which can be silenced via Python flags, functions, or an environment variable.

With `python -W ignore` for example, or with `export PYTHONWARNINGS="ignore"`

This is documented in further detail at 

https://docs.python.org/2/using/cmdline.html?highlight=pythonwarnings#cmdoption-w

And

https://docs.python.org/2/using/cmdline.html?highlight=pythonwarnings#envvar-PYTHONWARNINGS

The ceph-disk tool is going to get deprecated within the next two major releases (or so), so there is time for someone to get a `zfs` implementation if they choose to.

**Comment by wjwithagen** [comment]
@alfredodeza 
You are correct that it is going to take another 2 releases to depricate ceph-disk.
But I would really appriciate a FeeBSD conditional for now, as not to distract any FreeBSD testers.
It might give them the idea to try later.

**Pull request merged**